python/python/Ice/Communicator_local.py

```diff
@@ -0,0 +1,427 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) ZeroC, Inc. All rights reserved.
+#
+#
+# Ice version 3.7.10
+#
+# <auto-generated>
+#
+# Generated from file `Communicator.ice'
+#
+# Warning: do not edit this file.
+#
+# </auto-generated>
+#
+
+from sys import version_info as _version_info_
+import Ice, IcePy
+import Ice.LoggerF_local
```

Comment:
This _local is temporary too.

Note that we can just name these file Ice.LoggerF.py etc as it creates circular dependency errors.